cmd/collector/collector_test.go

```diff
@@ -0,0 +1,154 @@
+// Copyright 2024 VMware, Inc.
```

Comment:
Not related to this PR.
Should we use Vmware or Broadcom. I saw some projects on gitlab start use Broadcom instead?

Reply:
I can switch to Broadcom